docs/cloud/upandrunning.md

```diff
@@ -2,25 +2,31 @@
 
 In this guide, we will look at a minimal and quick way to get Prefect Cloud flow deployments up and running on your local machine. We will write a simple flow, build its Docker storage, deploy to Prefect Cloud, and orchestrate a run with the Local Agent. No extra infrastructure required!
 
+You might also want to look at the Running A Flow tutorial that is in the "Tutorials" section of the UI. (Accessed via the menu button at the top-left of the dashboard.)
+
 ### Prerequisites
 
-In order to start using Prefect Cloud we need to set up our authentication. Head to the UI and retrieve a `USER` API token which we will use to log into Prefect Cloud. We are also going to want to generate a `RUNNER` token and save that in a secure place because we will use it later when creating our Local Agent. 
+In order to start using Prefect Cloud we need to set up our authentication. Head to the UI and retrieve a `USER` API token from the user settings section (click on the button at the top left of the dashboard to find it). We will use the `USER` token to log into Prefect Cloud. We are also going to want to generate a `RUNNER` token (available in the team settings section of the UI) and save that in a secure place because we will use it later when creating our Local Agent.
 
-For information on how to create a `RUNNER` token visit the [Tokens](concepts/tokens.html) page.
+For more information on how to create a `USER` or `RUNNER` token visit the [Tokens](concepts/tokens.html) page.
 
 Let's use the Prefect CLI to log into Cloud. Run this command, replacing `$PREFECT_USER_TOKEN` with the `USER` token you generated a moment ago:
+
 ```
 $ prefect auth login --token $PREFECT_USER_TOKEN
 Login successful!
 ```
 
 Now you should be able to begin working with Prefect Cloud! Verify that you have a default project by running:
+
 ```
 $ prefect get projects
 NAME            FLOW COUNT  AGE         DESCRIPTION
 Hello, World!   0           1 day ago
 ```
 
+If you don't have a project set up, you can do so from the dashboard in the UI.
```

Comment:
Also from the CLI :) https://docs.prefect.io/cloud/concepts/cli.html#action-commands